pkg/plugins/hybrid/v1alpha/scaffolds/init.go

```diff
@@ -46,7 +46,8 @@ const (
 	hybridOperatorVersion = "0.0.1"
 
 	// helmPluginVersion is the operator-framework/helm-operator-plugin version to be used in the project
-	helmPluginVersion = "dc6c589504b2884e68dff6dc0e5e87ce8c24702f"
+	// This is the SHA of the latest commit on master, which will be used to scaffold the project.
```

Comment:
Does this mean every PR that merges to master needs to update this line?

Could we instead inject the value at build time similar to how we inject version information?

Reply:
See https://github.com/operator-framework/helm-operator-plugins/blob/dc6c589504b2884e68dff6dc0e5e87ce8c24702f/pkg/plugins/v1/scaffolds/init.go#L90-L95

Reply:
@joelanford This is helpful. But right now, the [tests](https://github.com/operator-framework/helm-operator-plugins/blob/e3bd72e9f2f8aa4b259b08b411028ade6cec097c/Makefile#L45) configured in Makefile build SDK locally and replace the plugin with the local path to test it out and it breaks since we don't import it as dependency in go.mod. Ill create an issue to track this in a follow up, so that we can make this modification and change the way test is written too.

Reply:
That seems reasonable for now.

But I'm curious if this needs to be resolved before the next release. I might not be fully understanding how this works, but it seems like it will never be possible for the commit SHA on this line to match the SHA of the commit that this line is in.

And if that's true, it means that when we go to tag this repo, this line will actually point to an older commit, and users will end up importing the library at that older commit.

I'm also curious if that test will actually test the current commit. It seems like it will always test the commit that's in this line?

Reply:
Hmm. Yeah, that's true. Can we let this be an input through operator-sdk where we provide the release to scaffold, till its used as a phase 1 plugin? This doesn't seem to be the ideal way to do so, I think.
But on a longer run even with phase 2 plugins this could be an issue because we have the library and plugin in the same place.
Yes, the test uses the current changes committed to scaffold and check if we are able to build an image. Thinking about it, how would we modify the tests to get the current commit and scaffold? If we cannot put the current commit at this place, the generated testdata and tests will always be verifying the previous commit.